cloud/deployment/inspect/inspect.go

```diff
@@ -0,0 +1,154 @@
+	if err != nil {
+		return err
+	}
+	fmt.Fprintln(out, string(infoToPrint))
+	return nil
+}
+
+func getDeploymentInspectInfo(sourceDeployment *astro.Deployment) (map[string]interface{}, error) {
+	var (
+		deploymentURL string
+		err           error
+	)
+
+	deploymentURL, err = deployment.GetDeploymentURL(sourceDeployment.ID, sourceDeployment.Workspace.ID)
+	if err != nil {
+		return nil, err
+	}
+	return map[string]interface{}{
+		"deployment_id":   sourceDeployment.ID,
+		"workspace_id":    sourceDeployment.Workspace.ID,
+		"cluster_id":      sourceDeployment.Cluster.ID,
+		"airflow_version": sourceDeployment.RuntimeRelease.AirflowVersion,
+		"release_name":    sourceDeployment.ReleaseName,
+		"deployment_url":  deploymentURL,
+		"webserver_url":   sourceDeployment.DeploymentSpec.Webserver.URL,
+		"created_at":      sourceDeployment.CreatedAt,
+		"updated_at":      sourceDeployment.UpdatedAt,
+		"status":          sourceDeployment.Status,
+	}, nil
+}
+
+func getDeploymentConfig(sourceDeployment *astro.Deployment) map[string]interface{} {
+	return map[string]interface{}{
+		"name":               sourceDeployment.Label,
+		"description":        sourceDeployment.Description,
+		"cluster_id":         sourceDeployment.Cluster.ID,
+		"runtime_version":    sourceDeployment.RuntimeRelease.Version,
+		"scheduler_au":       sourceDeployment.DeploymentSpec.Scheduler.AU,
+		"scheduler_replicas": sourceDeployment.DeploymentSpec.Scheduler.Replicas,
+	}
+}
+
+func getAdditional(sourceDeployment *astro.Deployment) map[string]interface{} {
+	return map[string]interface{}{
+		"alert_emails":         sourceDeployment.AlertEmails,
+		"worker_queues":        getQMap(sourceDeployment.WorkerQueues),
+		"astronomer_variables": getVariablesMap(sourceDeployment.DeploymentSpec.EnvironmentVariablesObjects), // API only returns values when !EnvironmentVariablesObject.isSecret
+	}
+}
+
+func getQMap(sourceDeploymentQs []astro.WorkerQueue) []map[string]interface{} {
+	queueMap := make([]map[string]interface{}, 0, len(sourceDeploymentQs))
+	for _, queue := range sourceDeploymentQs {
+		newQ := map[string]interface{}{
+			"id":                 queue.ID,
+			"name":               queue.Name,
+			"is_default":         queue.IsDefault,
+			"max_worker_count":   queue.MaxWorkerCount,
+			"min_worker_count":   queue.MinWorkerCount,
+			"worker_concurrency": queue.WorkerConcurrency,
+			"node_pool_id":       queue.NodePoolID,
+		}
+		queueMap = append(queueMap, newQ)
+	}
+	return queueMap
+}
+
+func getVariablesMap(sourceDeploymentVars []astro.EnvironmentVariablesObject) []map[string]interface{} {
+	variablesMap := make([]map[string]interface{}, 0, len(sourceDeploymentVars))
+	for _, variable := range sourceDeploymentVars {
+		newVar := map[string]interface{}{
+			"key":        variable.Key,
+			"value":      variable.Value,
+			"is_secret":  variable.IsSecret,
+			"updated-at": variable.UpdatedAt,
```

Comment:
```suggestion
			"updated_at": variable.UpdatedAt,
```
to keep field casing consistent

Reply:
Thank you! Almost had them all :)